tcp.go

```diff
@@ -44,6 +68,7 @@ func (ll *lingerListener) Accept() (manet.Conn, error) {
 		return nil, err
 	}
 	tryLinger(c, ll.sec)
+	tryKeepAlive(c, true)
```

Comment:
We should probably rename this struct.

Reply:
Which struct?

Reply:
GitHub didn't allow me to comment on that line: `lingerListener` -> `lingerAndKeepAliveListener` 😟

Reply:
Oh, I see. Yeah. Or just `tcpListener`. Up to you.

Reply:
That's better, thanks.